### DISCUSSION TITLE
Does Next13 support sever side fetching with Next Start yet?

### DISCUSSION THREAD

**Discussion by JohnVersus** [vercel/next.js#43410] [Help] [answered]
I recently tried updating my app to the next 13 and found that the output from the `next dev` and `next start` is different. My mainly relies on fetching data before loading and that seems to be not yet working with next13.

I checked the roadmap and did not find any details for it. 
https://beta.nextjs.org/docs/app-directory-roadmap#supported-and-planned-features

Is the `next start` supported yet for sever side fetching? If it is supported then I might be doing something wrong.

**Comment by icyJoseph** [comment, marked as answer]
There's "GSSP"-like, or forced-dynamic, behaviour. The default is to cache, but you can tell Next.js that certain pages, ought to be revalidated, and also generated as you go.

It is a whole different story if there seems to be a bit of a buggy behaviour. I have a page which uses `cookies` and that made it fully on-demand up to `13.0.4` where it stopped, ~~now in `13.0.5` I see it works fine again~~, and it still doesn't work :( 

- https://beta.nextjs.org/docs/api-reference/segment-config#dynamic
- https://beta.nextjs.org/docs/rendering/static-and-dynamic-rendering
- https://beta.nextjs.org/docs/upgrade-guide#server-side-rendering-getserversideprops

Read through those to get a better idea of the ideal production release behaviour. Do remember this is an open beta, and there'll be bugs and such, you are not expected to migrate your production apps to `appDir` during this period.

**Comment by JohnVersus** [comment]
Thanks for clarifying 🙏. I will look into some alternate ways.
